srf_generation/velocity_model_generation/q_model_generation/generate_all_qp_qs.py

```diff
@@ -70,6 +70,14 @@ def load_args():
         help="Path to the fault selection file.",
     )
 
+    parser.add_argument(
+        "-n",
+        "--n_processes",
+        type=int,
+        help="Number of Qp and Qs files to generate at once. At most one per fault/event.",
```

Comment:
Per rupture? Or per fault? Or per realisation?

Reply:
As the comment says, per fault.